### ISSUE TITLE
good bbox result, while bad mask result

### ISSUE THREAD

**Issue by longkun-uestc** [facebookresearch/maskrcnn-benchmark#1158]
## ❓ Questions and Help
when I trained maskrcnn on my own dataset, I got a good bbox result, but the quality of the mask was poor.  I converted the labels of my dataset into the standard format of the coco dataset, and then modified some of the configuration parameters.  Here are some parameters I changed when I run the code. The others are same as the parameters in maskrcnn-benchmark/config/defaults.py.

MODEL:
  META_ARCHITECTURE: "GeneralizedRCNN"
  WEIGHT: "catalog://ImageNetPretrained/MSRA/R-101"
  BACKBONE:
    CONV_BODY: "R-101-FPN"
  RESNETS:
    BACKBONE_OUT_CHANNELS: 256
  RPN:
    USE_FPN: True
    ANCHOR_STRIDE: (4, 8, 16, 32, 64) 
    ANCHOR_SIZES: (8, 16, 32, 64, 128)  # small anchor size
    PRE_NMS_TOP_N_TRAIN: 2000
    PRE_NMS_TOP_N_TEST: 1000
    POST_NMS_TOP_N_TEST: 1000
    FPN_POST_NMS_TOP_N_TEST: 2000  # 2 images per GPU
  ROI_HEADS:
    USE_FPN: True
  ROI_BOX_HEAD:
    POOLER_RESOLUTION: 7
    POOLER_SCALES: (0.25, 0.125, 0.0625, 0.03125)
    POOLER_SAMPLING_RATIO: 2
    FEATURE_EXTRACTOR: "FPN2MLPFeatureExtractor"
    PREDICTOR: "FPNPredictor"
    NUM_CLASSES: 2    # only two catogeries, the object to be detected and background
  ROI_MASK_HEAD:
    POOLER_SCALES: (0.25, 0.125, 0.0625, 0.03125)
    FEATURE_EXTRACTOR: "MaskRCNNFPNFeatureExtractor"
    PREDICTOR: "MaskRCNNC4Predictor"
    POOLER_RESOLUTION: 14
    POOLER_SAMPLING_RATIO: 2
    RESOLUTION: 28
    SHARE_BOX_FEATURE_EXTRACTOR: False
  MASK_ON: True
DATASETS:
  TRAIN: ("coco_2014_train", "coco_2014_valminusminival") # I rename my dataset as coco dataset
  TEST: ("coco_2014_minival",)
DATALOADER:
  SIZE_DIVISIBILITY: 32
SOLVER:
  BASE_LR: 0.001   # Smaller learning rate
  WEIGHT_DECAY: 0.0001
  STEPS: (40000, 60000)
  MAX_ITER: 80000
  CHECKPOINT_PERIOD: 2000
  IMS_PER_BATCH: 4                        # 2 GPU, 2 images per GPU
TEST:
  IMS_PER_BATCH: 2                        # 2 GPU, 1 image per GPU
OUTPUT_DIR: "./ckpoints/res_101_fpn"
INPUT:
  MIN_SIZE_TRAIN: 512                   # the size of our images is 512x512
  MAX_SIZE_TRAIN: 512
  MIN_SIZE_TEST: 512
  MAX_SIZE_TEST: 512

When I trained the model and tested on my test dataset, I got the following results:

Task: bbox
AP, AP50, AP75, APs, APm, APl
0.3825, 0.8268, 0.2799, 0.3478, 0.5447, 0.8000
Task: segm
AP, AP50, AP75, APs, APm, APl
0.0040, 0.0109, 0.0008, 0.0021, 0.0457, 0.0000

I got the normal bbox result, but the result of the segmentation is very poor.  I guess my parameter settings are incorrect， but I don't know how to set the parameters of the mask branch. Who can help me?

Here is my environment:
Ubuntu 18.4
cuda 9.0
cudnn 7.5.1
Graphics card：GeForce GTX 1080
pytorch: 1.3.0
python: 3.7

**Comment by longkun-uestc** [comment]
I have found the reason. I created the json label file incorrectly. The issue will be closed.

**Comment by lukelqy** [comment]
你好，我是华工一名研究生，也在做医学图像处理，能加个微信交流一下吗？wx：17779304585 感谢
